packages/hardhat-ethers/test/index.ts

```diff
@@ -21,6 +21,54 @@ describe("Ethers plugin", function () {
           ...Object.keys(ethers),
         ]);
       });
+
+      describe("Custom formatters", function () {
+        const customSymbol = Symbol.for("nodejs.util.inspect.custom");
+
+        const assertBigNumberFormat = function (
+          BigNumber: any,
+          value: string | number,
+          expected: string
+        ) {
+          assert.equal(BigNumber.from(value)[customSymbol](), expected);
```

Comment:
I think I'd do something like `util.format("%o", value)`, as that's what the console does.

Reply:
Nice, that is far better.